src/main/groovy/edu/wpi/first/gradlerio/frc/RoboRIO.groovy

```diff
@@ -109,8 +109,20 @@ class RoboRIO extends FRCCompatibleTarget {
     List<String> validImageVersions = []
 
     void verifyImageVersion(String image) {
-        if (!validImageVersions.contains(image))
+        def match = validImageVersions.find {
+            int index = it.indexOf('*')
+            if (index == -1) {
+                // no wildcard, check if versions are equal
+                return it == image
+            } else {
```

Comment:
this needs an `else if (index > image.size()) { return false }` otherwise the substring fails